documentation/404.md

```diff
@@ -0,0 +1,5 @@
+:orphan:
```

Comment:
Going by how the other directives work, I thought maybe

```{orphan}
```

was the Markdown corresponding to the `:orphan:` in rST.